tools/pom.xml

```diff
@@ -225,7 +225,12 @@
 					<artifactId>maven-surefire-plugin</artifactId>
 					<version>${maven-surefire-plugin-version}</version>
 					<configuration>
-						<argLine>-DCLDR_ENVIRONMENT=UNITTEST -Djava.awt.headless=true  -DCLDR_DIR=../../ -Xmx6g -enableassertions</argLine>
+					<systemPropertyVariables>
```

Comment:
The indentation seems off. Do we have a linter for that? More generally I've been wishing we had an agreed-on set of tools for normalizing spacing, etc., for all kinds of source files

Reply:
yes, i've fixed the indentation. It should be the same as Eclipse's now.

Agreed…